### PR TITLE
Hotfix/options screener fix

### PR DESCRIPTION
**Pull request by deeleeramone**
This patch is for #4250

Done:

- Restores the options screener
- Moves presets to Miscellaneous folder and updates the path
- Removes stale presets, adds new sample presets.
- Adjust columns being returned to be more intuitive.
- fixes tests
- adds options screener to SDK.

### CODE REVIEW

**Comment by jmaslek** [comment]
just an fyi -- make sure to mark as ready when you want me to look at

**Comment by deeleeramone** [comment]
> just an fyi -- make sure to mark as ready when you want me to look at

This wasn't supposed to have the realized volatility cones as part of this PR, but asides from the docs pages, it is ready to go now as well.

**Comment by jmaslek** [comment]
sdk was not properly generated: `AttributeError: 'TaRoot' object has no attribute 'hodges_tompkins'`

You need to run `python generate_sdk.py` .

Doing this results in errors (AttributeError: module 'openbb_terminal.common.technical_analysis.volatility_model' has no attribute 'rvol_hodges_tompkins').

Theres also an extra trailing , in economy.cpi that needs to be taken out to get the above error.

**Comment by jmaslek** [comment]
> sdk was not properly generated: `AttributeError: 'TaRoot' object has no attribute 'hodges_tompkins'`
> 
> You need to run `python generate_sdk.py` .
> 
> Doing this results in errors (AttributeError: module 'openbb_terminal.common.technical_analysis.volatility_model' has no attribute 'rvol_hodges_tompkins').
> 
> Theres also an extra trailing , in economy.cpi that needs to be taken out to get the above error.

i gotchu

**Comment by deeleeramone** [comment]
Now what's up with the tests? :(

File: tests/openbb_terminal/stocks/technical_analysis/test_volatility_model.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# IMPORTATION INTERNAL
+from openbb_terminal.common.technical_analysis import volatility_model
+
+# pylint: disable=W0621
+# pylint: disable=W0613
+
+MODELS = volatility_model.VOLATILITY_MODELS
+MOCK_DATA = pd.read_csv(
+    "tests/openbb_terminal/stocks/technical_analysis/csv/test_volatility_model/test_cones_df.csv",
```
Comment (jmaslek) [inline tests/openbb_terminal/stocks/technical_analysis/test_volatility_model.py:15]: This will break on windows

**Comment by jmaslek** [comment]
> Now what's up with the tests? :(

idk.  looking.  Also have no idea what you did to generate 200 test changes with newlines at the end

File: tests/openbb_terminal/stocks/technical_analysis/test_volatility_model.py (hunk 1 above)
Comment (deeleeramone) [reply tests/openbb_terminal/stocks/technical_analysis/test_volatility_model.py:15]: Which part breaks on Windows? Does it need to be called, `MOCK_STOCK_DF`?

from `test_ta_controller.py`:
```
MOCK_STOCK_DF = pd.read_csv(
    "tests/openbb_terminal/stocks/technical_analysis/csv/test_ta_controller/stock_df.csv",
    index_col=0,
)
```

Comment (jmaslek) [reply tests/openbb_terminal/stocks/technical_analysis/test_volatility_model.py:15]: windows doesnt read `/`.  Im fixing

**Review by jmaslek — APPROVED** [review]

**Pull request merged**